submissions/LuckyDnepr/Hooli-popup-task/styles-menu.css

```diff
@@ -0,0 +1,304 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+body {
+  width: 100%;
+  height: 100%;
+}
+
+html {
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
```

Comment:
Avoid styling using HTML tags. [Reason](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)

Reply:
Styling by HTML tags allowed for some complicated cases of when you set base styles.
The current case looks like a setting base style so don't change anything here)